runelite-client/src/main/java/net/runelite/client/plugins/friendnotes/FriendNotesConfig.java

```diff
@@ -0,0 +1,20 @@
+package net.runelite.client.plugins.friendnotes;
```

Comment:
The license header should be added to all new files.

Reply:
added